nixos/modules/programs/environment.nix

```diff
@@ -42,6 +43,7 @@ in
         GTK_PATH = [ "/lib/gtk-2.0" "/lib/gtk-3.0" ];
         XDG_CONFIG_DIRS = [ "/etc/xdg" ];
         XDG_DATA_DIRS = [ "/share" ];
+        XCURSOR_PATH = [ "/share/icons" ];
         MOZ_PLUGIN_PATH = [ "/lib/mozilla/plugins" ];
```

Comment:
Then again, I guess we stick all sorts of random mozilla and KDE stuff in here too... so perhaps this isn't the place to draw the line 😦